powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
Purpose of `ascii.json` theme is to use only characters which are valid in ASCII for use in environments where locale is not UTF-8. Switching to ascii theme *will not* cause powerline output only ASCII characters, only locale or relevant option (e.g. `&encoding` in Vim) can do such a thing. Just when you switch to ascii theme you will not get question marks for branch name prefixes because encoding unicode string to `latin1` (should locale be latin1) is a destructive operation.